### ISSUE TITLE
[Update docs] bundler: failed to load command: fastlane (/opt/hostedtoolcache/Ruby/3.0.0/x64/bin/fastlane)

### ISSUE THREAD

**Issue by reilem** [maierj/fastlane-action#64]
**EDIT:** to clear up some misunderstandings, this issue is primarily about fixing and updating the documentation. I am reporting this problem, and also have a solution for it. Specifying `ruby-version: '2.6.x'` in the setup-ruby action will fix the problem. However, the documentation still says `>= 2.6`. So I propose we change the documentation to include the ruby-version setup step using a version that will work.

------------

When I try to use this action with the ruby version recommended in the docs (>= 2.6) I get the following error:
```
2021-02-01T14:11:33.8007514Z ##[group]Run actions/setup-ruby@v1
2021-02-01T14:11:33.8008179Z with:
2021-02-01T14:11:33.8008775Z   ruby-version: >= 2.6
2021-02-01T14:11:33.8009361Z env:
2021-02-01T14:11:33.8009930Z   JAVA_VERSION: 8.0.181
2021-02-01T14:11:33.8010542Z   FLUTTER_VERSION: 1.22.3
2021-02-01T14:11:33.8011136Z   ***
2021-02-01T14:11:33.8011710Z   ***
2021-02-01T14:11:33.8012327Z   ***
2021-02-01T14:11:33.8012951Z   ***
2021-02-01T14:11:33.8013824Z   JAVA_HOME_8.0.181_x64: /opt/hostedtoolcache/jdk/8.0.181/x64
2021-02-01T14:11:33.8014651Z   JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.181/x64
2021-02-01T14:11:33.8015481Z   JAVA_HOME_8_0_181_X64: /opt/hostedtoolcache/jdk/8.0.181/x64
2021-02-01T14:11:33.8016371Z   FLUTTER_HOME: /opt/hostedtoolcache/flutter/1.22.3-stable/x64
2021-02-01T14:11:33.8017126Z   ***
2021-02-01T14:11:33.8017849Z   ***
2021-02-01T14:11:33.8018574Z ##[endgroup]
2021-02-01T14:11:33.8651996Z ##[group]Run maierj/fastlane-action@v2.0.0
2021-02-01T14:11:33.8652696Z with:
2021-02-01T14:11:33.8653256Z   lane: internal
2021-02-01T14:11:33.8653891Z   subdirectory: android
2021-02-01T14:11:33.8654495Z env:
2021-02-01T14:11:33.8655049Z   JAVA_VERSION: 8.0.181
2021-02-01T14:11:33.8655651Z   FLUTTER_VERSION: 1.22.3
2021-02-01T14:11:33.8656235Z   ***
2021-02-01T14:11:33.8656801Z   ***
2021-02-01T14:11:33.8657410Z   ***
2021-02-01T14:11:33.8658038Z   ***
2021-02-01T14:11:33.8658767Z   JAVA_HOME_8.0.181_x64: /opt/hostedtoolcache/jdk/8.0.181/x64
2021-02-01T14:11:33.8659592Z   JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.181/x64
2021-02-01T14:11:33.8660419Z   JAVA_HOME_8_0_181_X64: /opt/hostedtoolcache/jdk/8.0.181/x64
2021-02-01T14:11:33.8661299Z   FLUTTER_HOME: /opt/hostedtoolcache/flutter/1.22.3-stable/x64
2021-02-01T14:11:33.8662062Z   ***
2021-02-01T14:11:33.8662774Z   ***
2021-02-01T14:11:33.8663553Z   ***
2021-02-01T14:11:33.8664320Z   ***
2021-02-01T14:11:33.8687142Z   ***
2021-02-01T14:11:33.8687778Z ##[endgroup]
2021-02-01T14:11:33.9550504Z Executing lane internal on Linux.
2021-02-01T14:11:35.1995352Z Moving to subdirectory android
2021-02-01T14:11:42.8544474Z Fetching gem metadata from https://rubygems.org/.........
2021-02-01T14:11:43.2053297Z Fetching gem metadata from https://rubygems.org/.
2021-02-01T14:11:44.9840581Z Resolving dependencies........................................................................................
2021-02-01T14:11:45.0163677Z Fetching public_suffix 4.0.6
2021-02-01T14:11:45.0168844Z Fetching CFPropertyList 3.0.3
2021-02-01T14:11:45.0861356Z Installing CFPropertyList 3.0.3
2021-02-01T14:11:45.1314891Z Installing public_suffix 4.0.6
2021-02-01T14:11:45.1634928Z Fetching atomos 0.1.3
2021-02-01T14:11:45.1947624Z Fetching babosa 1.0.2
2021-02-01T14:11:45.2063946Z Installing atomos 0.1.3
2021-02-01T14:11:45.2327546Z Installing babosa 1.0.2
2021-02-01T14:11:45.2412912Z Using bundler 2.2.3
2021-02-01T14:11:45.2430923Z Fetching colored 1.2
2021-02-01T14:11:45.2788118Z Fetching highline 2.0.3
2021-02-01T14:11:45.2829048Z Installing colored 1.2
2021-02-01T14:11:45.3043264Z Fetching security 0.1.3
2021-02-01T14:11:45.3377994Z Installing highline 2.0.3
2021-02-01T14:11:45.3596302Z Installing security 0.1.3
2021-02-01T14:11:45.4089438Z Fetching excon 0.78.1
2021-02-01T14:11:45.4321677Z Fetching gh_inspector 1.1.3
2021-02-01T14:11:45.4747939Z Installing gh_inspector 1.1.3
2021-02-01T14:11:45.5097592Z Fetching json 1.8.6
2021-02-01T14:11:45.5128252Z Installing excon 0.78.1
2021-02-01T14:11:45.5577289Z Fetching multi_json 1.15.0
2021-02-01T14:11:45.6235038Z Installing json 1.8.6 with native extensions
2021-02-01T14:11:45.6641183Z Installing multi_json 1.15.0
2021-02-01T14:11:45.7303143Z Fetching plist 3.1.0
2021-02-01T14:11:45.7752507Z Installing plist 3.1.0
2021-02-01T14:11:45.8071097Z Fetching rubyzip 1.1.7
2021-02-01T14:11:45.8874174Z Installing rubyzip 1.1.7
2021-02-01T14:11:45.9469767Z Fetching terminal-table 1.4.5
2021-02-01T14:11:45.9775698Z Installing terminal-table 1.4.5
2021-02-01T14:11:46.0071791Z Fetching multipart-post 2.0.0
2021-02-01T14:11:46.0471388Z Installing multipart-post 2.0.0
2021-02-01T14:11:46.0729462Z Fetching unf_ext 0.0.7.7
2021-02-01T14:11:46.1438513Z Installing unf_ext 0.0.7.7 with native extensions
2021-02-01T14:11:49.1906675Z Fetching multi_xml 0.6.0
2021-02-01T14:11:49.2335641Z Installing multi_xml 0.6.0
2021-02-01T14:11:49.2604689Z Fetching claide 1.0.3
2021-02-01T14:11:49.2927239Z Installing claide 1.0.3
2021-02-01T14:11:49.3240483Z Fetching colored2 3.1.2
2021-02-01T14:11:49.3753735Z Installing colored2 3.1.2
2021-02-01T14:11:49.4022919Z Fetching dotenv 2.7.6
2021-02-01T14:11:49.4477040Z Installing dotenv 2.7.6
2021-02-01T14:11:49.4903240Z Fetching mini_magick 4.5.1
2021-02-01T14:11:49.5319332Z Installing mini_magick 4.5.1
2021-02-01T14:11:49.5642521Z Fetching rouge 2.0.7
2021-02-01T14:11:49.6425222Z Installing rouge 2.0.7
2021-02-01T14:11:49.7775796Z Fetching net-ssh 6.1.0
2021-02-01T14:11:49.8744047Z Installing net-ssh 6.1.0
2021-02-01T14:11:49.9484213Z Fetching slack-notifier 1.5.1
2021-02-01T14:11:49.9904865Z Installing slack-notifier 1.5.1
2021-02-01T14:11:50.0165115Z Fetching jwt 2.2.2
2021-02-01T14:11:50.0756902Z Installing jwt 2.2.2
2021-02-01T14:11:50.3738749Z Fetching memoist 0.16.2
2021-02-01T14:11:50.4161648Z Installing memoist 0.16.2
2021-02-01T14:11:50.4441623Z Fetching os 1.1.1
2021-02-01T14:11:50.4846212Z Installing os 1.1.1
2021-02-01T14:11:50.5157423Z Fetching httpclient 2.8.3
2021-02-01T14:11:50.6010254Z Installing httpclient 2.8.3
2021-02-01T14:11:50.6613910Z Fetching hurley 0.2
2021-02-01T14:11:50.6942740Z Installing hurley 0.2
2021-02-01T14:11:50.7294328Z Fetching mime-types-data 3.2020.1104
2021-02-01T14:11:50.8049325Z Fetching uber 0.0.15
2021-02-01T14:11:50.8177256Z Installing mime-types-data 3.2020.1104
2021-02-01T14:11:50.8377930Z Installing uber 0.0.15
2021-02-01T14:11:50.8876025Z Fetching retriable 2.1.0
2021-02-01T14:11:50.8889721Z Fetching thor 0.20.3
2021-02-01T14:11:50.9555772Z Installing retriable 2.1.0
2021-02-01T14:11:50.9617476Z Installing thor 0.20.3
2021-02-01T14:11:51.0304751Z Fetching terminal-notifier 1.6.3
2021-02-01T14:11:51.0651671Z Fetching nanaimo 0.3.0
2021-02-01T14:11:51.1140164Z Installing terminal-notifier 1.6.3
2021-02-01T14:11:51.1213610Z Installing nanaimo 0.3.0
2021-02-01T14:11:51.2003934Z Fetching addressable 2.7.0
2021-02-01T14:11:51.2367209Z Fetching commander 4.5.2
2021-02-01T14:11:51.2844568Z Installing commander 4.5.2
2021-02-01T14:11:51.2914129Z Installing addressable 2.7.0
2021-02-01T14:11:51.3721151Z Fetching faraday 0.17.3
2021-02-01T14:11:51.3730800Z Fetching xcpretty 0.3.0
2021-02-01T14:11:51.4581987Z Installing faraday 0.17.3
2021-02-01T14:11:51.4868640Z Installing xcpretty 0.3.0
2021-02-01T14:11:51.6095108Z Fetching net-sftp 2.1.2
2021-02-01T14:11:51.6664687Z Fetching unf 0.1.4
2021-02-01T14:11:51.7078187Z Installing net-sftp 2.1.2
2021-02-01T14:11:51.7649954Z Installing unf 0.1.4
2021-02-01T14:11:51.7763975Z Fetching mime-types 3.3.1
2021-02-01T14:11:51.8188960Z Installing mime-types 3.3.1
2021-02-01T14:11:51.8203541Z Fetching representable 2.3.0
2021-02-01T14:11:51.8645909Z Fetching xcodeproj 1.19.0
2021-02-01T14:11:51.8798467Z Installing representable 2.3.0
2021-02-01T14:11:51.9480838Z Installing xcodeproj 1.19.0
2021-02-01T14:11:51.9536704Z Fetching fastimage 1.6.8
2021-02-01T14:11:52.0095883Z Fetching credentials_manager 0.16.4
2021-02-01T14:11:52.0626413Z Installing credentials_manager 0.16.4
2021-02-01T14:11:52.1019288Z Fetching faraday_middleware 0.14.0
2021-02-01T14:11:52.1473323Z Installing faraday_middleware 0.14.0
2021-02-01T14:11:52.2309576Z Fetching signet 0.14.1
2021-02-01T14:11:52.2365911Z Installing fastimage 1.6.8
2021-02-01T14:11:52.3023171Z Installing signet 0.14.1
2021-02-01T14:11:52.3819928Z Fetching xcpretty-travis-formatter 1.0.1
2021-02-01T14:11:52.3831426Z Fetching domain_name 0.5.20190701
2021-02-01T14:11:52.4314635Z Installing xcpretty-travis-formatter 1.0.1
2021-02-01T14:11:52.4665559Z Installing domain_name 0.5.20190701
2021-02-01T14:11:52.4720445Z Fetching fastlane_core 0.59.0
2021-02-01T14:11:52.7376606Z Fetching krausefx-shenzhen 0.14.11
2021-02-01T14:11:52.7501795Z Installing fastlane_core 0.59.0
2021-02-01T14:11:52.8012388Z Fetching googleauth 0.15.0
2021-02-01T14:11:52.8135554Z Installing krausefx-shenzhen 0.14.11
2021-02-01T14:11:52.8624442Z Fetching http-cookie 1.0.3
2021-02-01T14:11:52.8644794Z Installing googleauth 0.15.0
2021-02-01T14:11:52.8989005Z Installing http-cookie 1.0.3
2021-02-01T14:11:52.9592349Z Fetching gym 1.12.1
2021-02-01T14:11:52.9598858Z Fetching scan 0.14.2
2021-02-01T14:11:52.9963082Z Installing gym 1.12.1
2021-02-01T14:11:53.0068956Z Installing scan 0.14.2
2021-02-01T14:11:53.0712482Z Fetching screengrab 0.5.6
2021-02-01T14:11:53.0722061Z Fetching snapshot 1.16.4
2021-02-01T14:11:53.1324788Z Installing screengrab 0.5.6
2021-02-01T14:11:53.1363770Z Installing snapshot 1.16.4
2021-02-01T14:11:53.2053491Z Fetching faraday-cookie_jar 0.0.7
2021-02-01T14:11:53.2064066Z Fetching google-api-client 0.9.4
2021-02-01T14:11:53.2573497Z Installing faraday-cookie_jar 0.0.7
2021-02-01T14:11:53.5307788Z Fetching spaceship 0.39.0
2021-02-01T14:11:53.5773237Z Installing spaceship 0.39.0
2021-02-01T14:11:53.6272758Z Installing google-api-client 0.9.4
2021-02-01T14:11:53.7146070Z Fetching cert 1.4.4
2021-02-01T14:11:53.7604386Z Installing cert 1.4.4
2021-02-01T14:11:53.8264639Z Fetching deliver 1.16.1
2021-02-01T14:11:53.8726886Z Installing deliver 1.16.1
2021-02-01T14:11:53.9602310Z Fetching sigh 1.12.1
2021-02-01T14:11:54.0131812Z Installing sigh 1.12.1
2021-02-01T14:11:54.0966277Z Fetching pem 1.4.1
2021-02-01T14:11:54.1437377Z Installing pem 1.4.1
2021-02-01T14:11:54.2194676Z Fetching pilot 1.12.1
2021-02-01T14:11:54.2835690Z Installing pilot 1.12.1
2021-02-01T14:11:54.3655288Z Fetching produce 1.3.2
2021-02-01T14:11:54.4100781Z Installing produce 1.3.2
2021-02-01T14:11:54.4256048Z Fetching xcode-install 1.4.0
2021-02-01T14:11:54.4641735Z Fetching frameit 2.8.0
2021-02-01T14:11:54.4827555Z Installing xcode-install 1.4.0
2021-02-01T14:11:54.5228383Z Installing frameit 2.8.0
2021-02-01T14:11:54.5570445Z Fetching match 0.11.1
2021-02-01T14:11:54.5959949Z Fetching supply 0.8.0
2021-02-01T14:11:54.6050113Z Installing match 0.11.1
2021-02-01T14:11:54.6551603Z Installing supply 0.8.0
2021-02-01T14:11:54.7031174Z Fetching fastlane 1.92.0
2021-02-01T14:11:54.8121799Z Installing fastlane 1.92.0
2021-02-01T14:11:55.7206235Z Bundle complete! 1 Gemfile dependency, 73 gems now installed.
2021-02-01T14:11:55.7208208Z Use `bundle info [gemname]` to see where a bundled gem is installed.
2021-02-01T14:11:55.9724039Z bundler: failed to load command: fastlane (/opt/hostedtoolcache/Ruby/3.0.0/x64/bin/fastlane)
2021-02-01T14:11:55.9769912Z /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core/pkg_file_analyser.rb:1:in `require': cannot load such file -- rexml/document (LoadError)
2021-02-01T14:11:55.9774172Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core/pkg_file_analyser.rb:1:in `<top (required)>'
2021-02-01T14:11:55.9775848Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core.rb:19:in `require'
2021-02-01T14:11:55.9777394Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core.rb:19:in `<top (required)>'
2021-02-01T14:11:55.9778915Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane-1.92.0/lib/fastlane.rb:1:in `require'
2021-02-01T14:11:55.9780418Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane-1.92.0/lib/fastlane.rb:1:in `<top (required)>'
2021-02-01T14:11:55.9782005Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane-1.92.0/lib/fastlane/cli_tools_distributor.rb:8:in `require'
2021-02-01T14:11:55.9783652Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane-1.92.0/lib/fastlane/cli_tools_distributor.rb:8:in `take_off'
2021-02-01T14:11:55.9785203Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/fastlane-1.92.0/bin/fastlane:5:in `<top (required)>'
2021-02-01T14:11:55.9786464Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/bin/fastlane:23:in `load'
2021-02-01T14:11:55.9787611Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/bin/fastlane:23:in `<top (required)>'
2021-02-01T14:11:55.9789009Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli/exec.rb:63:in `load'
2021-02-01T14:11:55.9790285Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli/exec.rb:63:in `kernel_load'
2021-02-01T14:11:55.9791536Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli/exec.rb:28:in `run'
2021-02-01T14:11:55.9792742Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli.rb:497:in `exec'
2021-02-01T14:11:55.9794118Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
2021-02-01T14:11:55.9795576Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
2021-02-01T14:11:55.9798643Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
2021-02-01T14:11:55.9799945Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli.rb:30:in `dispatch'
2021-02-01T14:11:55.9801245Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
2021-02-01T14:11:55.9802511Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli.rb:24:in `start'
2021-02-01T14:11:55.9803902Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/bundler-2.2.3/libexec/bundle:49:in `block in <top (required)>'
2021-02-01T14:11:55.9805389Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/friendly_errors.rb:130:in `with_friendly_errors'
2021-02-01T14:11:55.9806843Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/bundler-2.2.3/libexec/bundle:37:in `<top (required)>'
2021-02-01T14:11:55.9808094Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/bin/bundle:23:in `load'
2021-02-01T14:11:55.9809160Z 	from /opt/hostedtoolcache/Ruby/3.0.0/x64/bin/bundle:23:in `<main>'
2021-02-01T14:11:55.9856178Z ##[error]Error: Executing lane internal failed.
2021-02-01T14:11:55.9868551Z ##[error]Executing lane internal failed.
```

The relevant workflow part looks like this:
```
- uses: actions/setup-ruby@v1
  with:
    ruby-version: '>= 2.6'
  if: ***
- uses: maierj/fastlane-action@v2.0.0
  with:
    lane: 'internal'
    subdirectory: 'android'
  env:
    ***
  if: ***
```

Specifying `ruby-version: '2.6.x'` in the setup-ruby action does work. So we should update the docs to include this.

**Comment by Warns** [comment]
@maierj @KevinColemanInc I'm facing the same error as above. Please check my files below.

Directory structure
![image](https://user-images.githubusercontent.com/4021052/106490746-30819400-64c7-11eb-976f-1ecb5d94b46c.png)

Its basically as follows;
root
  - android
    - fastlane
  - ios
    - fastlane

```yaml
name: Deploy

on:
  push:
    branches: [ 3934 ]

jobs:
  distribute:
    runs-on: ubuntu-latest

    steps:

      - name: Checkout 
        uses: actions/checkout@v2

      - name: Setup Ruby
        uses: actions/setup-ruby@v1
        with:
          ruby-version: '>= 2.6'

      - name: Install Ruby bundle
        run: |
          bundle config path vendor/bundle
          bundle install --jobs 4 --retry 3

      - name: Configure Keystore
        run: |
          mkdir app
          echo "$ANDROID_KEYSTORE_FILE" | base64 -d > app/keystore.jks
          echo "storeFile=keystore.jks" >> keystore.properties
          echo "keyAlias=$KEYSTORE_KEY_ALIAS" >> keystore.properties
          echo "storePassword=$KEYSTORE_STORE_PASSWORD" >> keystore.properties
          echo "keyPassword=$KEYSTORE_KEY_PASSWORD" >> keystore.properties
        env:
          ANDROID_KEYSTORE_FILE: ${{ secrets.ANDROID_KEYSTORE_FILE }}
          KEYSTORE_KEY_ALIAS: ${{ secrets.KEYSTORE_KEY_ALIAS }}
          KEYSTORE_KEY_PASSWORD: ${{ secrets.KEYSTORE_KEY_PASSWORD }}
          KEYSTORE_STORE_PASSWORD: ${{ secrets.KEYSTORE_STORE_PASSWORD }}

      - name: Create Google Play Config file
        run: |
          echo "$GPLAY_CONFIG_JSON" > gplay_config.json.b64
          base64 -d -i gplay_config.json.b64 > gplay_config.json
        env:
          GPLAY_CONFIG_JSON: ${{ secrets.GPLAY_CONFIG_JSON }}

      - uses: maierj/fastlane-action@v2.0.0
        with:
          lane: 'beta'
          subdirectory: 'android'
```

Please guide us through this. Thanks!

The error log:
```
 Bundle complete! 1 Gemfile dependency, 73 gems now installed.
Bundled gems are installed into `./vendor/bundle`
bundler: failed to load command: fastlane (/home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/bin/fastlane)
/home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core/pkg_file_analyser.rb:1:in `require': cannot load such file -- rexml/document (LoadError)
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core/pkg_file_analyser.rb:1:in `<top (required)>'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core.rb:19:in `require'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane_core-0.59.0/lib/fastlane_core.rb:19:in `<top (required)>'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane-1.92.0/lib/fastlane.rb:1:in `require'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane-1.92.0/lib/fastlane.rb:1:in `<top (required)>'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane-1.92.0/lib/fastlane/cli_tools_distributor.rb:8:in `require'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane-1.92.0/lib/fastlane/cli_tools_distributor.rb:8:in `take_off'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/gems/fastlane-1.92.0/bin/fastlane:5:in `<top (required)>'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/bin/fastlane:23:in `load'
	from /home/runner/work/***-mobile-test/***-mobile-test/android/vendor/bundle/ruby/3.0.0/bin/fastlane:23:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli/exec.rb:63:in `load'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli/exec.rb:63:in `kernel_load'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli/exec.rb:28:in `run'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli.rb:497:in `exec'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli.rb:30:in `dispatch'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/cli.rb:24:in `start'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/bundler-2.2.3/libexec/bundle:49:in `block in <top (required)>'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/3.0.0/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/lib/ruby/gems/3.0.0/gems/bundler-2.2.3/libexec/bundle:37:in `<top (required)>'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/bin/bundle:23:in `load'
	from /opt/hostedtoolcache/Ruby/3.0.0/x64/bin/bundle:23:in `<main>'
Error: Error: Executing lane beta failed.
Error: Executing lane beta failed.
```

**Comment by KevinColemanInc** [comment]
Are you using ruby 3.0? I haven't started using this yet so I don't know how much advice I can give. sorry

**Comment by reilem** [comment]
@Warns Specify `ruby-version: '2.6.x'` in the setup-ruby action. That should solve your problem.

By specifying `>= 2.6` in the docs you are allowing ruby to go to whatever version higher than or including 2.6. And I think very recently the update for 3.0.0 happened (I do not use ruby for anything other than CI tools so I could be mistaken). So now lots of users of this action are going to start seeing failed deployments.

**Comment by Warns** [comment]
Based on the latest note in the readme file I changed the ruby tag, probably misunderstood it. Indeed it passed that step by specifying the version as `'2.6'`

**Comment by reilem** [comment]
@Warns Don't worry, you did not misunderstand anything 😄 You followed the documentation perfectly, unfortunately the documentation is wrong. That is why I made this issue, to notify the maintainer that they need to update their documentation.

**Comment by maierj** [comment]
Hi guys,
thanks for creating this issue.
I conducted a few tests in the test project that belongs to this action (https://github.com/maierj/fastlane-action-test). I added different Ruby versions to the build matrix (2.5.8, 2.6.6, 2.7.2, 3.0.0) and let the test workflow run for each of them: turns out that only the runs with ruby-version set to 3.0.0 fail (https://github.com/maierj/fastlane-action-test/actions/runs/541194115). I also found out that fastlane is not yet supporting Ruby 3 (https://github.com/fastlane/fastlane/issues/17931).
The proposed ruby-version number in the documentation of this action was not wrong when it was published, since Ruby 3 wasn't available at that time (it was released on 2020-12-25), but it's wrong now, since it chooses the latest Ruby version available (3.0.0) when you specify ">=2.6.0".

I'll change the documentation to recommend the latest Ruby version before 3.0.0, which is 2.7.2. I'll also add a hint that fastlane doesn't support Ruby 3 yet.
